sql/parser2/eval.go

```diff
@@ -108,8 +117,33 @@ func (d dstring) String() string {
 	return string(d)
 }
 
+type dtuple []Datum
+
+func (d dtuple) Bool() (dbool, error) {
+	return false, fmt.Errorf("cannot convert NULL to bool")
```

Comment:
Oops. Think I had a merge conflict with the multiple branches I've been working in.